5_databases/week3/Additional-queries.sql

```diff
@@ -0,0 +1,44 @@
+use homework3;
+--1 Get meals that has a price smaller than a specific price fx 90--
+SELECT *
+FROM `Meal`
+WHERE price < 90;
+
+--2 Get meals that still has available reservations--
+SELECT Meal.*
+FROM Meal JOIN Reservation  ON Meal.id = Reservation.meal_id
+WHERE  Reservation.created_date > NOW();
+
+--3  Get meals that partially match a title. Rød grød med will match the meal with the title Rød grød med fløde-
+SELECT *
+FROM Meal
+WHERE title LIKE 'Rød grød med%';
+
+--4 meals between two date--
+SELECT *
+FROM Meal
+WHERE created_date BETWEEN '2023-01-01' AND '2023-12-31';
+
+--5 get limit of  number --
+SELECT *
+FROM Meal
+WHERE created_date BETWEEN '2023-01-01' AND '2023-12-31'
+LIMIT 1;
```

Comment:
Here we just ask you to get a specific number of rows. You can remove the WHERE clause.